pkg/roachprod/vm/aws/aws.go

```diff
@@ -1151,11 +1188,15 @@ func (p *Provider) listRegion(
 				RemoteUser:             opts.RemoteUserName,
 				VPC:                    in.VpcID,
 				MachineType:            in.InstanceType,
-				CPUArch:                vm.ParseArch(in.Architecture),
```

Comment:
I wonder if `GetVMArchitecture` is really needed in this case, since we already know it. There could be some edge cases where `describe-instances` doesn't populate it, but we also don't know if `describe-instance-types` is going to succeed.